### DISCUSSION TITLE
Multiple and LazyMultiple options

### DISCUSSION THREAD

**Discussion by Nerdman4U** [75lb/command-line-args#131] [Show and tell]
Hello,
I pushed fix for issue #106! There is a simple test included. Any comments? 
I am new to github workflow so please let me know if i am missing something...  :)
Nerdman4U

**Comment by 75lb** [comment]
Hi, you pushed the fix where? To a local fork? Please submit a Pull Request if you want your work to be reviewed.. Cheers.

**Comment by Nerdman4U** [comment]
Hi! Sure! Sorry i did not recognize your reply, i am not yet familiar with the process... learning...

**Comment by Nerdman4U** [comment]
Hey, as far as i understand, i have committed pull request here: https://github.com/75lb/command-line-args/pull/132.
